app/controllers/requests_controller.rb

```diff
@@ -34,6 +34,12 @@ def cancel
     redirect_back(fallback_location: root_path)
   end
 
+  def delete
+    @request.deleted!
```

Comment:
Yes, I thought that once, but it is too dangerous that deleting real database record.
So, I set "delete" status to requests and receivers.
How do you think about it?